cli.js

```diff
@@ -129,5 +132,9 @@ shareDBDoc.subscribe(() => {
 
 server.listen(port, () => {
   console.log(`Editor is live at http://localhost:${port}`);
-  open(`http://localhost:${port}`);
+  (async function () {
+    await ngrok.authtoken('2NSu2nKIuMoHIc5z9XDPIgm2gpz_7oDwUferEeadw1Mvc66Qi');
```

Comment:
This API key is, I would guess, tied to your personal NGrok account? I would suggest we do not add this to the repo. If there is a strong need to use an API key, it should probably go into an environment variable.